Source/SwiftLintFramework/Extensions/Dictionary+SwiftLint.swift

```diff
@@ -151,6 +151,50 @@ public struct SourceKittenDictionary {
     }
 }
 
+extension SourceKittenDictionary {
+    /// Traversing all substuctures of the dictionary hierarchically, calling |traverseBlock| on each node.
```

Comment:
Nit: use markdown-style code markup using backticks instead of `|` so it renders nicely in Xcode.

```suggestion
    /// Traversing all substructures of the dictionary hierarchically, calling `traverseBlock` on each node.
```